styles.css

```diff
@@ -1 +1,76 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+* {
+    padding: 0;
+    margin: 0;
+}
+
+:root {
+    --item-color: hsl(353, 63%, 41%);
+    --bg-container: hsl(35, 95%, 91%);
+    --text-color: white;
+}
+
+.container {
+    height: 800px;
+    padding: 16px;
+    background-color: var(--bg-container);
+    display: grid;
+    row-gap: 16px;
+    column-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:    "header-left"
+                            "aside-left"
+                            "header-large"
+                            "aside-right"
+                            "main"
+                            "."
+                            "footer";
+}
+
+@media screen and (min-width: 576px) {
+    .container {
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-template-rows: 50px 190px 100px auto 50px;
+        grid-template-areas: "header-left header-large header-large"
```

Comment:
34 t/m 39: Oplossing werkt goed, ik had iets vergelijkbaars qua benaming, in de uitwerking staat het zo: 
`grid-template-areas:
                "a c c"
                "b c c"
                "b e d"
                "b e ."
                "f f f";`
En dat vind ik wel beter visueel leesbaar.